nautobot_design_builder/design_job.py

```diff
@@ -160,69 +168,88 @@ def implement_design(self, context, design_file, commit):
         """
         design = self.render_design(context, design_file)
         self.log_debug(f"New Design to be implemented: {design}")
-        deprecated_design = {}
 
         # The design to apply will take into account the previous journal that keeps track (in the builder_output)
         # of the design used (i.e., the YAML) including the Nautobot IDs that will help to reference them
-        self.builder.builder_output[design_file] = copy.deepcopy(design)
-        last_journal = (
-            self.builder.journal.design_journal.design_instance.journals.filter(active=True)
-            .exclude(id=self.builder.journal.design_journal.id)
-            .exclude(builder_output={})
-            .order_by("-last_updated")
-            .first()
-        )
-        if last_journal and last_journal.builder_output:
-            # The last design output is used as the reference to understand what needs to be changed
-            # The design output store the whole set of attributes, not only the ones taken into account
-            # in the implementation
-            previous_design = last_journal.builder_output[design_file]
-            self.log_debug(f"Design from previous Journal: {previous_design}")
+        # self.environment.builder_output[design_file] = copy.deepcopy(design)
```

Comment:
I guess this is no longer relevant?

Reply:
Correct, the builder_output has been completely removed from the code base as it is not required for the functionality. I will remove this commented out line.